### PR TITLE
Fixed brightness control; Reduced number of brightness levels to 4.

### PR DESCRIPTION
**Pull request by orcinus**
I've reduced the number of brightness levels to 4 (16 is a bit much) and changed the way brightness PWM is calculated.

Originally, at max brightness, it was actually at half brightness (2^15 = 32768 instead of 65536) because you were simply shifting a 1 by 15 (1000000000000000). All the other levels were scaled accordingly. So i've instead used 2^16 as a starting point, and then it gets shifted RIGHT the more, the lower the level.

However, even that's not quite ideal, because the first levels are nearly useless, because perceived brightness isn't quite linear, so i've fudged it up a bit.

Bottom line, with these changes, max brightness truly is 100% PWM, while other levels are squished slightly towards the max instead of spaced linearly.

### CODE REVIEW

**Comment by jackhumbert** [comment]
Awesome! 4 levels makes sense. Thanks for contributing!
